CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_greenhouse v0.8.0
+[PR #28](https://github.com/fivetran/dbt_greenhouse/pull/28) is a breaking change due to [upstream updates](
+https://github.com/fivetran/dbt_greenhouse_source/releases/tag/v0.8.0):
+
+## Breaking Changes
+- Casted `*_id` fields as strings in upstream `stg_*` models to ensure compatibility in downstream joins.
```

Comment:
Similar note from the [source PR notes](https://github.com/fivetran/dbt_greenhouse_source/pull/31#discussion_r1852992296), can you callout that the majority of these ID fields were initially integers and that they should expect the downstream fields in the end models will now be strings as opposed to integers.